### ISSUE TITLE
Enabling Query Sync - Not showing Run test and finder not searching for .bazelproject

### ISSUE THREAD

**Issue by QAInsights** [bazelbuild/intellij#7107]
### Description of the bug:

Enabling Query Sync: The option to run tests (inline editor)is not appearing, and the finder is unable to search for the `.bazelproject` file.

### Which category does this issue belong to?

Intellij

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

1. Enable Query Sync
2. Import .bazelproject
3. Run button (in inline editor) is not appearing for unit tests
4. Searching for `.bazelproject` is not showing up in the IntelliJ finder

### Which Intellij IDE are you using? Please provide the specific version.

IntelliJ IDEA 2024.3 (Ultimate Edition) Build #IU-243.21565.193, built on November 12, 2024

### What programming languages and tools are you using? Please provide specific versions.

Java

### What Bazel plugin version are you using?

2024.11.19.0.1-api-version-243

### Have you found anything relevant by searching the web?

No

**Comment by dkashyn-sfdc** [comment]
There is a good/guaranteed chance that https://github.com/bazelbuild/intellij/pull/7081/files fixes the tests.

**Comment by tpasternak** [comment]
Yeah, it might be caused by the same thing as #7081 , it will be available in the next beta (this week)
